CMakeLists.txt

```diff
@@ -112,9 +112,10 @@ option(DISABLE_TTF "Disable support for TTF provided by freetype2.")
 option(ENABLE_SCRIPTING "Enable script / plugin support." ON)
 option(ENABLE_ASAN "Enable the AddressSanitizer.")
 option(ENABLE_UBSAN "Enable the UndefinedBehaviourSanitizer.")
-
+option(ENABLE_HEADERS_CHECK "Check if include directives in header files are correct. Only works with clang" ON)
```

Comment:
Should really prefix project specific cmake vars with the project name (but i know we don't do that anywhere else)

Reply:
We may as well default this to off. The vast majority of users won't want it on its only really used by the ci

Reply:
Good point

Reply:
done